static/editor.js

```diff
@@ -83,6 +83,7 @@ define(function (require) {
             value: state.source || defaultSrc || "",
             scrollBeyondLastLine: false,
             language: cmMode,
+            fontFamily: 'Fira Mono',
```

Comment:
Can you make the same change in the `compiler.js` and `diff.js` views too? I think they should all use the same font.